Microsoft.Toolkit.Uwp.Input.GazeControls/GazeFilePicker.xaml

```diff
@@ -0,0 +1,501 @@
+﻿<ContentDialog
```

Comment:
We haven't extended a ContentDialog before, so will have to think about approach here. We should at least follow the guidance on inheriting from the WinUI styles: https://docs.microsoft.com/en-us/uwp/api/Windows.UI.Xaml.Controls.ContentDialog?view=winrt-20348#derived-controls-with-winui-styles

Reply:
Thanks. I will take a closer look and update.

Reply:
Fixed with the latest commit.

Reply:
Yeah, I think we can still make this a templated control as well. See the pattern that the existing ContentDialog change uses as well from WinUI: https://github.com/microsoft/microsoft-ui-xaml/blob/main/dev/CommonStyles/ContentDialog_themeresources.xaml